packages/superset-ui-core/test/callApi/rejectAfterTimeout.test.js

```diff
@@ -0,0 +1,20 @@
+/* eslint promise/no-callback-in-promise: 'off' */
+import rejectAfterTimeout from '../../src/callApi/rejectAfterTimeout';
+import throwIfCalled from '../utils/throwIfCalled';
+
+describe('rejectAfterTimeout', () => {
```

Comment:
nit
Add `()` in `describe('rejectAfterTimeout()'`) so we know it is a function when reading test result